src/modules/layout/containers/__tests__/LocationLayout.spec.js

```diff
@@ -42,28 +41,34 @@ describe('LocationLayout', () => {
 
   describe('connect', () => {
     const locationsEndpoint = new EndpointBuilder('locations')
-      .withUrl('https://weird-endpoint/api.json')
+      .withRouterToUrlMapper(() => 'https://weird-endpoint/api.json')
       .withMapper(json => json)
       .withResponseOverride(locations)
       .build()
 
-    const mockStore = configureMockStore([thunk])
+    const location = 'augsburg'
+    const path = '/:location/:language'
 
-    const store = mockStore({
-      locations: new Payload(false),
-      router: {params: {location: 'augsburg', language: 'en', id: '1234'}, route: '/:location/:language'}
+    const store = createReduxStore(createHistory, {
+      router: {params: {location: location, language: language}, route: path}
     })
 
     test('should map state to props', () => {
-      const tree = mount(
+      const locationLayout = mount(
         <Provider store={store}>
           <EndpointProvider endpoints={[locationsEndpoint]}>
-            <ConnectedLocationLayout><MockNode /></ConnectedLocationLayout>
+            <ConnectedLocationLayout />
           </EndpointProvider>
         </Provider>
-      )
-      // todo: add locations
-      expect(tree.find(ConnectedLocationLayout).childAt(0).props()).toMatchSnapshot()
+      ).find(LocationLayout)
+
+      expect(locationLayout.props()).toEqual({
+        path: path,
+        location: location,
+        language: language,
+        locations: locations,
+        dispatch: expect.any(Function)
```

Comment:
does anyone now why there is this dispatch prop in the props?

Reply:
Every connected component gets the dispatch prop